### PR TITLE
chore(KtRadio/KtRadioGroup): remove them

### PR DESCRIPTION
**Pull request by carsoli**
no longer used, deprecated since a while

### CODE REVIEW

File: packages/kotti-ui/source/kotti-style/forms/_misc.scss [hunk 1]
```diff
@@ -73,10 +73,7 @@
 }
 
 // Disabled fields
-.form-input,
-.form-checkbox,
-.form-switch,
-.form-radio {
```
Comment (FlorianWendelborn) [inline packages/kotti-ui/source/kotti-style/forms/_misc.scss:79]: This class is still used in `src/user-panel-frontend/src/components/basket/wizard/BasketWizardPostProcessingSingleSelect.vue`

**Review by FlorianWendelborn — DISMISSED** [review]

File: packages/documentation/data/menu.ts [hunk 2]
```diff
@@ -19,7 +19,6 @@ import {
 	KtNavbar,
 	KtPagination,
 	KtPopover,
-	KtRadio,
```
Comment (FlorianWendelborn) [inline packages/documentation/data/menu.ts:22]: Please mark the 4 relevant tasks as complete in #273. After this PR, we only have `KtInput` to remove, then we can finally release Kotti 3 officially 🚀

Comment (FlorianWendelborn) [reply packages/documentation/data/menu.ts:22]: Also, `KtInput` only appears 5 times in the codebase by now. So, the more challenging issue will be the associated classes.

Comment (FlorianWendelborn) [reply packages/documentation/data/menu.ts:22]: Oh, I just noticed we forgot to add the inline edit to that document. But still, some good progress towards V3.

Comment (carsoli) [reply packages/documentation/data/menu.ts:22]: the inline edit is also wip by me now, would be great to take a look when I submit the final changes today so I can also merge the new one :)

**Review by HansAuger — APPROVED** [review]

**Pull request merged**